docs/management/index-lifecycle-policies/example-index-lifecycle-policy.asciidoc

```diff
@@ -1,23 +1,167 @@
 [role="xpack"]
+
 [[example-using-index-lifecycle-policy]]
-=== Example of using an index lifecycle policy
+=== Tutorial:  Use ILM to manage Filebeat time-based indices
+
+With index lifecycle management (ILM), you can create policies to define actions
+that are automatically performed against indices as they age.  In this tutorial,
+you will use {kib}’s *Index Lifecycle Policies* to modify and create ILM policies.
```

Comment:
> With index lifecycle management (ILM), you can create policies to define actions that are automatically performed against indices as they age.

This implies that ILM only works based on age, but we can also take actions based on index size or number of docs. More importantly, this doesn't really address the _why_ of ILM. It helps you automatically manage your indices for performance and retention. You may want to consider reusing a sentence from https://www.elastic.co/guide/en/elasticsearch/reference/master/overview-index-lifecycle-management.html.

> In this tutorial, you will use Kibana’s Index Lifecycle Policies to modify and create ILM policies.

Nit: I'd reword to avoid future tense if possible. Something like:

```
This tutorial shows you how to use use {kib}'s *Index Lifecycle Policies* to
create and change {ilm-cap} policies.
```

The bolded **Index Lifecycle Policies** seemed unnecessary to me, but maybe that's just a Kibana docs style.

Reply:
Yep, it's a Kibana docs style.